generators/server/templates/src/test/java/package/cucumber/stepdefs/CucumberContextConfiguration.java.ejs

```diff
@@ -0,0 +1,38 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%=packageName%>.cucumber.stepdefs;
+
+import <%=packageName%>.<%= mainClass %>;
+import cucumber.api.java.Before;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+
+@SpringBootTest
+@AutoConfigureMockMvc
```

Comment:
I'm wondering: why have you changed the `@WebAppConfiguration` annotation to `@AutoConfigureMockMvc`?

Reply:
I took that from the fix suggested here:
https://github.com/cucumber/cucumber-jvm/blob/master/examples/spring-txn/src/test/java/cucumber/examples/spring/txn/CucumberContextConfiguration.java

I don't know enough about the difference of the two and their possible impact on testing JHipster code.

Reply:
I tested it with @WebAppConfiguration and the tests still pass so I'll change it back to @WebAppConfiguration.